reactjs-films-homework/src/components/MovieCard/GenresList/GenresList.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import styles from './GenresList.module.scss';
+
+interface IGenresList {
+  genreName: string[];
+}
+
+const GenresList: React.FC<IGenresList> = ({ genreName }) => {
+  return (
+    <p className={styles.genre}>
+      {genreName &&
```

Comment:
What do you need this check? As I see 'genreName' type is an array of strings. So that it's always truthy.

Reply:
This redundant check was removed.
I completely understood this mistake.